src/main/scala/Harness.scala

```diff
@@ -4,9 +4,10 @@ import firrtl._
 import firrtl.ir._
 
 import java.io.Writer
+import scala.collection.mutable.ArrayBuffer
 
 object HarnessGenerator {
-  def harnessConnections(m: Module) = {
+  def ections(m: Module) = {
```

Comment:
I can't tell if GitHub truncated the name in this view, but it appears to be now named `ections`. Is the intent of that clear?

Reply:
My bad, this looks like a mistake.

Reply:
```suggestion
  def harnessConnections(m: Module) = {
```

Reply:
```suggestion
  def harnessConnections(m: Module) = {
```